src/tarkov-data-manager/data/changed_quests.json

```diff
@@ -635,5 +652,92 @@
         "propertiesChanged" : {
             "minPlayerLevel": 47
         }
+    },
+    "60e71c48c1bfa3050473b8e5": {
+        "name": "Crisis",
+        "propertiesChanged" : {
+            "minPlayerLevel": 48
+        }
+    },
+    "5eaaaa7c93afa0558f3b5a1c": {
+        "name": "The Survivalist Path - Junkie",
+        "taskRequirementsRemoved": [
+            {
+                "id": "5d25d2c186f77443e35162e5",
+                "name": "The Survivalist Path - Cold Blooded"
+            }
+        ]
+    },
+    "6179b3a12153c15e937d52bc": {
+        "name": "Energy Crisis",
+        "propertiesChanged" : {
+            "minPlayerLevel": 25
+        }
+    },
+    "6089736efa70fc097863b8f6": {
+        "name": "Back Door",
+        "propertiesChanged" : {
+            "taskRequirements": [
+                {
+                    "task": "5ac3477486f7741d651d6885",
+                    "name": "Scout",
+                    "status": [
+                        "complete"
+                    ]
+                }
+            ]
+        }
+    },
+    "6089732b59b92115597ad789": {
+        "name": "Surplus GOods",
```

Comment:
Typ**O**?

Reply:
Yeah, but the names aren't critical. They're only used to make it human-readable.